packages/plugin-js/src/index.ts

```diff
@@ -12,7 +12,7 @@ export interface Options {
   js?: boolean | string;
   /** output JSON? (default: false) */
   json?: boolean | string;
-  /** output meta? (default: false) */
+  /** include meta object in output? (default: true) */
```

Comment:
So this technically isn't true. There is no default for `meta` set anywhere, so the true default is `undefined`. But [the code explicitly checks for `false`](https://github.com/drwpow/cobalt-ui/blob/main/packages/plugin-js/src/index.ts#L161) so `undefined` has the same behavior as `true`. Given the option is semantically a boolean, I think its most helpful to say it defaults to `true` here and not bother developers with implementation details.

Reply:
Good improvement!